proposals/0000-bytearray-literals.rst

```diff
@@ -0,0 +1,345 @@
+.. proposal-number:: Leave blank. This will be filled in when the proposal is
+                     accepted.
+
+.. trac-ticket:: Leave blank. This will eventually be filled with the Trac
+                 ticket number which will track the progress of the
+                 implementation of the feature.
+
+.. implemented:: Leave blank. This will be filled in with the first GHC version which
+                 implements the described feature.
+
+.. highlight:: haskell
+
+This proposal is `discussed at this pull request <https://github.com/ghc-proposals/ghc-proposals/pull/134>`_.
+
+.. contents::
+
+ByteArray# literals
+===================
+
+This is a proposal to introduce ``ByteArray#`` and ``(# Word#, Addr# #)``
+literals, and slightly change ``Addr#`` literals. In short you'll be able
+to write
+
+.. code-block:: haskell
+
+  "Literals"#b           -- ByteArray#
+  "\xef\xbb\xbf"#abytes  -- Addr#
+  "Юникод"#ucp1251       -- (# Int#, Addr# #)
+
+additionally to current
+
+.. code-block:: haskell
+
+  "primitive"#           -- Addr# in utf8
+  "string                -- String or IsString a => a
```

Comment:
No quotes?

Reply:
Thanks for spotting. Correct others as well.